src/gtest/test_mempool.cpp

```diff
@@ -83,7 +83,8 @@ TEST(Mempool, PriorityStatsDoNotCrash) {
     unsigned int nHeight = 92045;
     double dPriority = view.GetPriority(tx, nHeight);
 
-    CTxMemPoolEntry entry(tx, nFees, nTime, dPriority, nHeight, true);
+    // TODO: SPROUT_BRANCH_ID
```

Comment:
This constant is defined in #2925; once that merges I'll adjust the final commit here to use it (which will work because Homu will merge this branch into master before testing it).

Reply:
Oh, except if there are merge conflicts, then I'll have to rebase anyway **shrug**

Reply:
Actually it's not defined the #2925 `main.h` anymore as I ended up not needing it as a function parameter default (instead height is being passed).

Reply:
ACK; I've re-defined it in the SigHash PR, but here I will probably just use the full string, and we can reduce it later.

Reply:
I realised in my pairing just now with @daira that this PR has the full version of a partial commit back-ported in #2903. So I'll bring `SPROUT_BRANCH_ID` into this PR so we can merge it first, and then rebase #2903 on it.